core/src/tokens.ts

```diff
@@ -19,6 +19,9 @@ export const getTokenDecimalsBySymbol = function (symbol: string): number {
     if (tokenName === 'DAI') {
         return DAI_NUMBER_OF_DIGITS;
     }
+    if (tokenName === 'MCD_GOV') {
```

Comment:
I also suggest to use `MKR` here and everywhere, eg by extending `getTokenAddressByNetworkAndSymbol`